css-contain-2/Overview.bs

```diff
@@ -1681,6 +1681,10 @@ Suppressing An Element's Contents Entirely: the 'content-visibility' property {#
 			content-visibility/content-visibility-with-top-layer-005.html
 			content-visibility/content-visibility-with-top-layer-006.html
 			</wpt>
+
+		* The element or one of its ancestors participates in an ongoing {{ViewTransition}},
```

Comment:
This can be simplified to: "The element's [involved in a view transition](https://drafts.csswg.org/css-view-transitions-1/#involved-in-a-view-transition) bit is true".

That is meant to track the exact state we need here and makes it more explicit when this state changes.